.pre-commit-hooks.yaml

```
@@ -0,0 +1,10 @@
+-   id: panvimdoc
+    name: pandoc to vimdoc
+    description: Convert markdown documentation to vimdoc (using docker)
+    entry: /panvimdoc.pre-commit.sh
+    language: docker
+    files: ^README\.md$
+    args:
+    - --input-file
+    - README.md
```

Does this need to be indented one more?

```suggestion
    args:
        - '--input-file'
        - README.md
```